src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const BASE_COST = 40;
+
+  if (days < 3) {
+    return days * BASE_COST;
+  } else if (days < 7) {
+    return days * BASE_COST - 20;
+  } else {
+    return days * BASE_COST - 50;
+  }
```

Comment:
Avoid using literals directly in the expressions. Define constants for the discount values and the threshold for the number of days that qualify for those discounts. This will make your code more readable and maintainable.